### PR TITLE
Show correct language when node is opened

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?148701

Are there any (performance) reasons to be more surgical about this, rather than just refreshing the whole tree?

### CODE REVIEW

**Comment by czue** [comment]
(quite likely yes)

File: src/core.js [hunk 1]
```diff
@@ -841,6 +841,7 @@ define([
             _this.activateQuestionTypeGroup(mug);
         }).bind("open_node.jstree", function (e, data) {
             _this.activateQuestionTypeGroup(_this.data.core.form.getMugByUFID(data.node.id));
+            _this.refreshVisibleData();
```
Comment (czue) [inline src/core.js]: how often does this event fire?

Comment (emord) [inline src/core.js]: Should only be called when a node is opened from being collapsed

**Comment by emord** [comment]
I don't think there will be a huge performance hit as it's only occurring on opening a node. Could also only update the children nodes of the mug being opened though.

This seems like it would be a good automated test to write though.

**Pull request merged**